compiler/llvm/clangUtil.cpp

```diff
@@ -4144,6 +4144,9 @@ static void makeBinaryLLVMForCUDA(const std::string& artifactFilename,
     }
   }
 
+  // avoid warning about not statically knowing the stack size when recursive
+  // functions are called from the kernel
+  ptxasFlags += " --suppress-stack-size-warning ";
```

Comment:
Do we need to add a similar flag for AMD?

Reply:
We don't seem to need it for AMD.